specutils/io/default_loaders/wcs_fits.py

```diff
@@ -157,12 +246,16 @@ def wcs1d_fits_writer(spectrum, file_name, hdu=0, update_header=False, **kwargs)
         raise ValueError(f'Only Spectrum1D objects with valid WCS can be written as wcs1d: {err}')
```

Comment:
I'm not _that_ familiar with the workings of WCS (and especially `wcs.to_fits()`) - is there any realistic scenario for setting `hdu` to a value other than 0, given that the initial `hdulist` is always created from the spectrum's WCS anyway? I guess what I'm asking is, is there really utility in having empty HDUs in front of the specified `hdu` index? There are no options to specify the HDU numbers for uncertainty and mask, so it's not like the user could write out in order [uncertainty, mask, flux], for example. Maybe we just get rid of the `hdu` argument altogether?

Reply:
As long as this is not considered "public" API and ok to break just like that, I agree.

Reply:
> I'm not _that_ familiar with the workings of WCS (and especially `wcs.to_fits()`) - is there any realistic scenario for setting `hdu` to a value other than 0, given that the initial `hdulist` is always created from the spectrum's WCS anyway?

It's not as much associated to using a WCS, but rather to the flux being stored in an ImageHDU, which can be the PrimaryHDU or an extension, while tables can only be stored in extensions. However for symmetry reasons or whatever sometimes writing the image to an extension, too, and using just a basic PrimaryHDU, is desired (there was one comment somewhere sketching just this scenario, but I cannot find it either in the Slack discussion or any related issue now).
I think originally accepting any value for `hdu` was just introduced as a poor substitute for not being able to write multiple spectra to one file/hdulist, but there is probably no real use case for this. For `hdu=1` I'd say there is, and this should should still remain available.

Reply:
Thanks for the explanation...I guess it makes sense to leave it in then for now.